CHANGELOG.md

```diff
@@ -25,6 +30,7 @@
 - `[*]` [**BREAKING**] Drop support for Node 6 ([#8455](https://github.com/facebook/jest/pull/8455))
 - `[*]` Add Node 12 to CI ([#8411](https://github.com/facebook/jest/pull/8411))
 - `[*]` [**BREAKING**] Upgrade to Micromatch v4 ([#8852](https://github.com/facebook/jest/pull/8852))
+- `[*]` Disable `valid-typeof`, `no-undef` for ts overrides ([#8382](https://github.com/facebook/jest/pull/8382))
```

Comment:
`valid-typeof` includes `bigint` starting in 6.0.0-rc.0 see comment in `.eslintrc.js`